app/src/androidTestDeviceDebug/kotlin/de/digitalService/useID/PinEntryFieldTest.kt

```diff
@@ -41,7 +42,9 @@ class PinEntryFieldTest {
                 onDone = mockOnDone
             )
 
-            focusRequester.requestFocus()
+            LaunchedEffect(true){
+                focusRequester.requestFocus()
```

Comment:
These tests all failed with "FocusRequester is not initialized. [...] Are you attempting to request focus during composition? Focus requests should be made in response to some event. Eg Modifier.clickable { focusRequester.requestFocus() }". Production code looks ok.